website/content/docs/nia/installation/install.mdx

```diff
@@ -9,36 +9,88 @@ description: >-
 
 Refer to the [introduction](https://learn.hashicorp.com/tutorials/consul/consul-terraform-sync-intro?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial for details about installing, configuring, and running Consul-Terraform-Sync on your local machine with the Terraform driver.
 
-## Installing Consul-Terraform-Sync
+## Install Consul-Terraform-Sync
 
-To install Consul-Terraform-Sync, find the [appropriate package](https://releases.hashicorp.com/consul-terraform-sync/) for your system and download it as a zip archive. Unzip the package to extract the binary named consul-terraform-sync. Move the consul-terraform-sync binary to a location available on your PATH.
+<Tabs>
+<Tab heading="Pre-compiled binary">
+
+To install Consul-Terraform-Sync, find the [appropriate package](https://releases.hashicorp.com/consul-terraform-sync/) for your system and download it as a zip archive. Unzip the package to extract the binary named `consul-terraform-sync`. Move the `consul-terraform-sync` binary to a location available on your `PATH`.
+
+For the CTS Enterprise binary, download a zip archive with the `+ent` metadata. [CTS Enterprise requires a Consul Enterpise license](/docs/nia/enterprise/license) to run.
+
+Example:
+```shell-session
+$ echo $PATH
+/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
+$ mv ./consul-terraform-sync /usr/local/bin/consul-terraform-sync
+```
+
+Once installed, verify the installation works by prompting the `-version` or `-help` option.
+
+```shell-session
+$ consul-terraform-sync -version
+```
+
+</Tab>
+<Tab heading="Docker">
+
+Install and run Consul-Terraform-Sync as a [Docker container](https://hub.docker.com/r/hashicorp/consul-terraform-sync).
+
+For the CTS Enterprise, use the Docker image [`hashicorp/consul-terraform-sync-enterprise`](https://hub.docker.com/r/hashicorp/consul-terraform-sync-enterprise).
+
+```shell-session
+$ docker pull hashicorp/consul-terraform-sync
+```
+
+Once installed, verify the installation works by prompting the `-version` or `-help` option.
+
+```shell-session
+$ docker run --rm hashicorp/consul-terraform-sync -version
+```
+
+</Tab>
+<Tab heading="Homewbrew on OS X">
+
+The CTS OSS binary is available in the HashiCorp tap, a repository of all our Homebrew packages.
```

Comment:
Do we not have an enterprise binary available on Homebrew? Maybe something to call out explicitly.

Reply:
From what I could find it is not supported